src/m365/teams/commands/channel/channel-remove.ts

```diff
@@ -143,12 +132,50 @@ class TeamsChannelRemoveCommand extends GraphCommand {
         default: false,
         message: `Are you sure you want to remove the channel ${channelName} from team ${args.options.teamId}?`
```

Comment:
maybe we could update the message if the user would specify the `teamName` it would be cool to see the name here not the id. The user might not be aware of the actual `teamId` when she or he is passing the name. 
what do you think ?